### ISSUE TITLE
Have an API call that will render a PNG of a graph

### ISSUE THREAD

**Issue by RobertJBarron** [skydive-project/skydive#1889]
Currently, the only way to share a skydive graph requires the user to login to skydive and render the graph again in realtime.

This makes is more difficult to share a specific graph easily.

Use cases - embed the png in a mail or in a chat platform like Slack.
Example of use in another application:
https://play.grafana.org/render/d/000000029/prometheus-demo-dashboard?orgId=1&refresh=5m
vs
https://play.grafana.org/d/000000029/prometheus-demo-dashboard?orgId=1&refresh=5m

**Comment by lebauce** [comment]
There is a way to retrieve the graph using the DOT format :
`skydive client query --format dot G > graph.dot`

Then, you can use `graphviz` to convert it to a PNG :
`dot -Tpng graph.dot -o outfile.png`

However, it won't look as (nice as ?) with the WebUI

**Comment by RobertJBarron** [comment]
Exactly :)

You'd lose a lot with this back door method.

**Comment by eranra** [comment]
@lebauce can we use iFrame and Skydive launch-in-context ? 

(https://community.grafana.com/t/text-box-iframe/11663)

BTW: For new versions of grafana need to configure: https://grafana.com/docs/installation/configuration/#disable-sanitize-html
(so this will not work out of the box with play.grafana.org which uses latest grafana)

**Comment by safchain** [comment]
Another approach could be to use the topology export/import

```
SKYDIVE_ANALYZER=127.0.0.1:8082 skydive client topology export > skydive.json
SKYDIVE_ANALYZER=127.0.0.1:8082 skydive client topology import --file skydive.json
```

You can import to an external analyzer. All topology data will be import and one can use the WebUI.

**Comment by hunchback** [comment]
@RobertJBarron - does the above provide a reasonable solution? if so then please close issue

**Comment by RobertJBarron** [comment]
Not really. I'm asking for an API that will return an image in realtime, not export the data that then needs to be massaged/visualized by a 3rd party/offline.
